ElementX/Sources/Screens/InviteUsersScreen/InviteUsersScreenModels.swift

```diff
@@ -41,12 +41,10 @@ struct InviteUsersScreenViewState: BindableState {
     var selectedUsers: [UserProfileProxy] = []
     var membershipState: [String: MembershipState] = .init()
     
-    var isSearching: Bool {
-        !bindings.searchQuery.isEmpty
-    }
+    var isSearching = false
     
     var hasEmptySearchResults: Bool {
-        isSearching && usersSection.type == .searchResult && usersSection.users.isEmpty
+        !bindings.searchQuery.isEmpty && usersSection.type == .searchResult && usersSection.users.isEmpty
```

Comment:
```suggestion
        !isSearching && !bindings.searchQuery.isEmpty && usersSection.type == .searchResult && usersSection.users.isEmpty
```
Shouldn't it be like this?

Reply:
Well I didn't want to change the original logic so I copied over what the old `isSearching` used to do ¯\_(ツ)_/¯

Reply:
Got it! :D
I think we use it to show the "empty screen view". So maybe it's better to show it after the search ends?

Reply:
Okay cool, I'll give it a shot then